src/components/Header.jsx

```diff
@@ -1,26 +1,44 @@
-import { useContext } from "react"
 import { UserContext } from "../contexts/User"
-import { useEffect, useState } from "react"
+import { useEffect, useState, useContext } from "react"
 import { Logo, NavBar, Users } from "."
 import { getTopics } from "../utils/api"
-import { menu } from "../utils/menu"
-import HamburgerOpen from "../assets/hamburguer-open.svg"
+import Hamburger from "../assets/hamburguer.svg"
 import Cross from "../assets/cross.svg"
 
 export const Header = () => {
-    const [topics, setTopics] = useState(["Coding", "Cooking", "Football"])
-
-	const { user } = useContext(UserContext)
+	const [menu, setMenu] = useState(false)
+    const [topics, setTopics] = useState([
+		{
+			slug: "coding",
+			description: "Code is love, code is life"
+		},
+		{
+			slug: "football",
+			description: "FOOTIE!"
+		},
+		{
+			slug: "cooking",
+			description: "Hey good looking, what you got cooking?"
+		}
+	])
+	const { user, setUser } = useContext(UserContext)
 	
      useEffect(() => {
         getTopics().then((result) => {
             setTopics(result)
         })
-     }, [])
+    }, [])
```

Comment:
By initialising the state to a hardcoded list - if down the line there are changed to lists of topics (for example, if one is deleted, or some are added), the initial incorrect list may be confusing for the user.
Could you please instead use a loading state to show the list of topics are being fetched 📩